packages/@aws-cdk-testing/framework-integ/test/aws-elasticloadbalancingv2/test/integ.alb.log.imported-bucket.js.snapshot/aws-cdk-alb-log-imported-bucket-integ.template.json

```diff
@@ -438,28 +438,64 @@
        "Principal": {
         "Service": "delivery.logs.amazonaws.com"
        },
+       "Resource": [
+        {
+         "Fn::Join": [
+          "",
+          [
+           "arn:aws:s3:::",
+           {
+            "Ref": "Bucket83908E77"
+           },
+           "/prefix-connection-log/AWSLogs/",
+           {
+            "Ref": "AWS::AccountId"
+           },
+           "/*"
+          ]
+         ]
+        },
+        {
+         "Fn::Join": [
+          "",
+          [
+           "arn:aws:s3:::",
+           {
+            "Ref": "Bucket83908E77"
+           },
+           "/prefix/AWSLogs/",
+           {
+            "Ref": "AWS::AccountId"
+           },
+           "/*"
+          ]
+         ]
+        }
+       ]
+      },
+      {
+       "Action": "s3:GetBucketAcl",
+       "Effect": "Allow",
+       "Principal": {
+        "Service": "delivery.logs.amazonaws.com"
+       },
        "Resource": {
         "Fn::Join": [
          "",
          [
           "arn:aws:s3:::",
           {
            "Ref": "Bucket83908E77"
-          },
-          "/prefix/AWSLogs/",
-          {
-           "Ref": "AWS::AccountId"
-          },
-          "/*"
+          }
          ]
         ]
        }
       },
       {
-       "Action": "s3:GetBucketAcl",
+       "Action": "s3:PutObject",
        "Effect": "Allow",
        "Principal": {
-        "Service": "delivery.logs.amazonaws.com"
+        "AWS": "arn:aws:iam::797873946194:root"
```

Comment:
is this expected ?

Reply:
The diff is hard to review, but this is what I expected.
Originally, the policy allowing s3:GetBucketAcl was the final element in the Statement array, but the policy allowing s3:PutObject from the ALB AWS account is now the final element.